src/nvim/api/vim.c

```diff
@@ -1562,3 +1563,15 @@ Object nvim_get_proc(Integer pid, Error *err)
 #endif
   return rvobj;
 }
+
+Integer nvim_get_caps_state(Error *err )
+  FUNC_API_SINCE(4)
+{
+  int res = capslock_status();
+  if (res == -1) {
+    api_set_error(err, kErrorTypeException,
+                  "Unable to find the Capslock State");
```

Comment:
Why “find” here? “Determine” sounds better, especially since you are searching something only in case of linux.

Reply:
Ok sounds better